test/testframe.c

```diff
@@ -497,31 +515,26 @@ TestSummary(FILE *stream)
     fprintf(stream, "\n\n");
 }
 
-/*
- * Perform test cleanup
- */
-void
-TestCleanup(void)
```

Comment:
`TestCleanup()` was only used in a few places and is covered entirely by `PerformTests()` now